specs/protocol/deputy-pause-module.md

```diff
@@ -104,28 +96,27 @@ replaced.
 > typically only rely on a subset of the following assumptions. Different invariants may rely on
 > different assumptions. Refer to individual invariants for their dependencies.
 
-### aSCP-001: Superchain-wide pause authorization process is well-defined
+### aSCP-001: Pause authorization process is well-defined
```

Comment:
SCP = superchain pause? maybe we should have a legend for these acronyms

Reply:
Oh we should kill that and just always refer to the Pause Mechanism and link it.

Reply:
Oh sorry I misread this

Reply:
Going to keep as-is for now, this is relatively experimental, the actual acronyms don't matter that much for now. If we use this more then I'll figure out a better naming scheme